core/src/epicli/data/common/ansible/playbooks/roles/logstash/tasks/main.yml

```diff
@@ -1,24 +1,33 @@
 ---
-- name: Install Logstash package
+- name: Install OS packages
   package:
     name: "{{ _packages[ansible_os_family] }}"
     state: present
   vars:
     _packages:
+      # Bundled with ElasticSearch OpenJDK 15 doesn't work correctly with Logstash
+      # as there are startup options that are deprecated in OpenJDK 15 such as UseConcMarkSweepGC
       Debian:
+        - openjdk-8-jre-headless
```

Comment:
Java 8 is quite old, have you tried 11?

Reply:
THen we whould update this accross all installs we do, not only for ARM I think.

Reply:
No, I just used the package that is available in repository. Will try 11.

Reply:
We may also use Java 8 here - since we have only this version in epirepo and create a separate task to replace Java 8 with 11 for all Java based components (including upgrades).  It would be nice to still have only one version in epirepo - if possible.